jenkins/pipeline-parallel-mtr.groovy

```diff
@@ -238,7 +238,9 @@ void setupTestSuitesSplit() {
             wget \${RAW_VERSION_LINK}/${BRANCH}/mysql-test/mysql-test-run.pl -O ${WORKSPACE}/mysql-test-run.pl
             chmod +x ${WORKSPACE}/suites-groups.sh
             set +e
-            ${WORKSPACE}/suites-groups.sh check ${WORKSPACE}/mysql-test-run.pl
+            # Set WORKER_x_MTR_SUITES and check if suites list is consistent with the one specified in mysql-test-run.pl
+            source ${WORKSPACE}/suites-groups.sh
+            check_suites ${CMAKE_BUILD_TYPE} ${WORKSPACE}/mysql-test-run.pl
```

Comment:
Check should not depend on build type, right? We always use the same set of suites.

Reply:
Did you read the topic of this PR?
```
PS-8978: Use different WORKER_x_MTR_SUITES for Debug and RelWithDebInfo
```

Reply:
`check_suites` calls `get_suites` to define `WORKER_x_MTR_SUITES` so it needs `${CMAKE_BUILD_TYPE}`,